test/kwc-nav-item_test.html

```diff
@@ -19,26 +19,27 @@
     </test-fixture>
     <test-fixture id="default">
         <template>
-            <kwc-nav-item label="Nav Item"></kwc-nav-item>
+            <kwc-nav-item>[[label]]</kwc-nav-item>
         </template>
     </test-fixture>
     <test-fixture id="icon">
         <template>
-            <kwc-nav-item label="Nav Item" icon-id="challenges"></kwc-nav-item>
+            <kwc-nav-item icon-id="challenges">Nav Item</kwc-nav-item>
         </template>
     </test-fixture>
     <test-fixture id="active">
         <template>
-            <kwc-nav-item label="Nav Item" active="true"></kwc-nav-item>
+            <kwc-nav-item class="iron-selected">Nav Item</kwc-nav-item>
         </template>
     </test-fixture>
     <test-fixture id="icon-and-active">
         <template>
-            <kwc-nav-item label="Nav Item" icon-id="challenges" active="true"></kwc-nav-item>
+            <kwc-nav-item class="iron-selected" icon-id="challenges">Nav Item</kwc-nav-item>
         </template>
     </test-fixture>
 
     <script>
+        var defaultLabel = 'Nav Item';
```

Comment:
Does this get used anywhere? I can't find it if so, but maybe I'm being blind...

Reply:
Nope! Removing!